### PR TITLE
[DN.dk] New ruleset

### PR DESCRIPTION
**Pull request by fuglede**
Most references seem to be securable, but all content pages redirect to HTTP.

### CODE REVIEW

**Comment by J0WI** [comment]
fordel and historie are also refused.
The [Dd]efault.aspx is optional (but https://www.dn.dk/?ID=7244 also redirects to http)

I would mark the rule as partial since it only loads resources over https.

**Comment by fuglede** [comment]
Agreed on all accounts.

File: src/chrome/content/rules/DN.dk.xml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+		- historie ¹
+		- tur ¹
+
+	¹: Refused
+-->
+<ruleset name="DN.dk (partial)">
+	<target host="dn.dk" />
+	<target host="www.dn.dk" />
+
+	<exclusion pattern="^http://www\.dn\.dk/([dD]efault\.aspx|\?).*" />
```
Comment (J0WI) [inline src/chrome/content/rules/DN.dk.xml]: This also matches ```http://www.dn.dk/default.aspxfoo```. Do we care about it?

Comment (fuglede) [inline src/chrome/content/rules/DN.dk.xml]: I didn't come across any parts of the site where that would be a problem, at least.

**Pull request merged**